doc/feature/rescaleOutput.md

```diff
@@ -0,0 +1,78 @@
+Rescaling the resulting volume to the output data type range
+============================================================
+
+Since Fivox makes all the computations in floating point values, and supports
+different output types in the voxelize tool (just uint8 in the Livre
+DataSource), we need a way to correctly rescale the resulting values, based
+on the input data range and the output type range.
+
+## Requirements
+
+* The internal computations should always be done in floating point values.
+* The data source for Livre must always convert the resulting volume from
+  float to uint8_t values.
+* The voxelize tool will always write a volume in the specified output data
+  type, converting it from float if needed.
+* The process of rescaling the volume data should always consider the output
+  min and max values (known for each data type, e.g. 0-255 for uint8_t), and
+  the input window (min and max values present in the input data).
```

Comment:
min and max values present in the input data or derived from the input data? LFP does sum, so your float voxel max might be greater than the input max?